config/locales/shared/da.yml

```diff
@@ -0,0 +1,173 @@
+---
+da:
```

Comment:
Why add the translations here when they are in the added `rails-i18n` gem?

Reply:
For most apps that I've worked on we would have to go to rails-i18n repository to lookup/fetch default translations.
If you would want to change a default translation you would need to know the translation key and then you would have to go to rails-i18n repo to look it up which would be quite annoying. 
Having translations extracted from gem makes it more convenient to edit and also helps developers to understand how we structure translations.

Reply:
Valid arguments.
However, if some translations are improved upstream we don't get it.
So I prefer only overwriting the translations, when needed.

@jeppester What do you think?